rc/extra/gdb.kak

```diff
@@ -0,0 +1,283 @@
+            }
+            /\*running/ {
+                send("gdb-clear-location")
+            }
+            /\*stopped/ { 
+                file = get($0, "fullname=\"", "[^\"]*", "\"")
+                line = get($0, "line=\"", "[0-9]+", "\"")
+                send("gdb-handle-stopped " line " \"" file "\"")
+            }
+            /=breakpoint-created/ {
+                id = get($0, "number=\"", "[0-9]+", "\"")
+                enabled = get($0, "enabled=\"", "[yn]", "\"")
+                line = get($0, "line=\"", "[0-9]+", "\"")
+                file = get($0, "fullname=\"", "[^\"]*", "\"")
+                send("gdb-handle-breakpoint-created " id " " enabled " " line " \"" file "\"")
+            }
+            /=breakpoint-modified/ {
+                id = get($0, "number=\"", "[0-9]+", "\"")
+                enabled = get($0, "enabled=\"", "[yn]", "\"")
+                line = get($0, "line=\"", "[0-9]+", "\"")
+                file = get($0, "fullname=\"", "[^\"]*", "\"")
+                send("gdb-handle-breakpoint-modified " id " " enabled " " line " \"" file "\"")
+            }
+            /=breakpoint-deleted/ {
+                id = get($0, "id=\"", "[0-9]+", "\"")
+                send("gdb-handle-breakpoint-deleted " id)
+            }
+            /&"print/ {
+                printing = 1
+                print_value = get($0, "print ", ".*", "..\"") " == "
+            }
+            /~".*"/ {
+                if (printing == 1) {
+                    print_value = print_value get($0, "= ", ".*", "\"")
+                    printing = 2
+                } else if (printing) {
+                    print_value = print_value "\n" get($0, "\"", ".*","\"")
+                }
+            }
+            /\^done/ {
+                if (printing) {
+                    # trim trailing \n
+                    print_value = substr(print_value, 0, length(print_value) - 3)
+                    # QUOTE => \\QUOTE
+                    gsub("'\''", "\\\\'\''", print_value)
+                    # eval -client $client QUOTE info  -- \QUOTE $string \QUOTE QUOTE
+                    send("eval -client '"$kak_client"' '\''info -- \\'\''" print_value "\\'\'\''")
+                    printing = 0
+                }
+            }
+            add-highlighter ref gdb
+            '
+        } 2>/dev/null >/dev/null &
+        printf "$!" > "${tmpdir}/pid"
+        printf "set-option global gdb_dir %s\n" "$tmpdir"
+        printf "info \"%s\"\n" "Please instruct gdb to \\\"new-ui mi3 ${tmpdir}/pty\\\""
+    }
+    hook -group gdb global BufOpenFile .* %{
+        gdb-refresh-location-flag
+        gdb-refresh-breakpoints-flags %val{buffile}
+    }
+    hook -group gdb global KakEnd .* %{
+        gdb-stop
+    }
+}
+
+define-command gdb-stop %{
+    try %{
+        %sh{
+            if [ -n "$kak_opt_gdb_dir" ]; then
+                pkill -P $(cat "${kak_opt_gdb_dir}/pid")
```

Comment:
This command isn't provided on some systems, is there anyway to emulate its effects using a plain `kill`?